src/AstTransforms.cpp

```diff
@@ -240,17 +352,17 @@ bool MaterializeAggregationQueriesTransformer::materializeAggregationQueries(
 
             // -- create a new clause --
 
-            auto relName = "__agg_rel_" + toString(counter++);
+            auto relName = "__agg_body_rel_" + toString(counter++);
```

Comment:
Haven't you just added a function for this?

Reply:
Yes, but the function is specific for __agg_rel relations. I wanted to be able to distinguish between what relations were encapsulating an entire aggregate (__agg_rel_) and which ones were just there to materialize the aggregate body (__agg_body_rel_). I could maybe make the function generic for name